editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -531,18 +535,18 @@ const RowForArrayControl = React.memo((props: RowForArrayControlProps) => {
   return (
     <React.Fragment>
       {when(dataPickerButtonData.popupIsOpen, dataPickerButtonData.DataPickerComponent)}
-      <InspectorSectionHeader>
+      <div css={hoverBackgroundStyle}>
         <SimpleFlexRow
           style={{ gap: 5, justifyContent: 'space-between', flexGrow: 1, paddingRight: 3 }}
         >
-          <FlexRow style={{ gap: 5 }} ref={dataPickerButtonData.setReferenceElement}>
-            <PropertyLabel
-              target={[propPath]}
-              style={{ textTransform: 'capitalize', paddingTop: 2 }}
-            >
+          <FlexRow
+            style={{ flex: 1, gap: 5, justifyContent: 'space-between' }}
+            ref={dataPickerButtonData.setReferenceElement}
+          >
+            <PropertyLabel target={[propPath]} style={objectPropertyLabelStyle}>
```

Comment:
I unified the look and feel of object and array sections, working on both of them highlighted just how different they looked, and it felt wildly offputting

Reply:
That's good but I can see this drifting again. I wonder if we can extract a lot of this into a shared component to prevent that